query-languages/m/table-expandlistcolumn.md

```diff
@@ -27,12 +27,28 @@ Table.ExpandListColumn(
 )
 ```
 
+## Example 2
+
+Split the nested table column [Components] in the table.
```

Comment:
Nix "in the table", both here and above in the existing example.

```suggestion
Split the nested table column [Components].
```

Reply:
Good idea. Done.